src/xml2csv.py

```diff
@@ -1,90 +1,81 @@
-import csv
-import gc
 import json
-import os
 import sys
+from csv import DictWriter
 from datetime import datetime
-from tempfile import TemporaryDirectory
+from io import StringIO
 
-import numpy as np
 from bs4 import BeautifulSoup
```

Comment:
Just noticed: since you are not using BeautifulSoup anymore, you can remove this `import` and its requirement in the [conda_requirements.txt](https://github.com/datasciencebr/serenata-de-amor/blob/934db02ec33ddc8958ec320f0a130904f643ba3e/conda_requirements.txt) file.